### PR TITLE
Add re-using->reusing

### PR DESCRIPTION
**Pull request by korverdev**

### CODE REVIEW

**Comment by korverdev** [comment]
Something seems to be up with the code coverage check. This PR is just adding more misspellings, so shouldn't impact code coverage one way or the other.

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -45303,6 +45303,7 @@ re-uplods->re-uploads
 re-usable->reusable
 re-use->reuse
 re-useable->reusable
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:45305]: While w're at it, I suggest adding:
```
re-used->reused
re-uses->reuses
```

**Comment by DimitriPapadopoulos** [comment]
About the CI error, I guess codecov has been updated under the hood, changing the metrics and the results at the project level. Hopefully they'll push a new version soon that will fix the bug.

**Comment by DimitriPapadopoulos** [comment]
@korverdev Could you rebase against master? We forced a merge in another PR, hopefully resetting project-level coverage metrics.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (korverdev) [reply codespell_lib/data/dictionary.txt]: I've done so. Thanks for the suggestion!

**Comment by korverdev** [comment]
@DimitriPapadopoulos Accidentally did a merge instead, but that seems to have cleared it up. Thanks for your help!

**Comment by korverdev** [comment]
I stand corrected. Looks like that check took a bit longer than expected to appear. I'll close this PR and reopen off a new branch later today.

**Pull request closed without merging**